src/network/networkd-network.c

```diff
@@ -488,43 +488,90 @@ int network_load_one(Manager *manager, const char *filename) {
                 log_warning_errno(r, "%s: Failed to add default route on device, ignoring: %m",
                                   network->filename);
 
-        r = ordered_hashmap_ensure_allocated(&manager->networks, &string_hash_ops);
+        struct stat stats;
+        if (stat(filename, &stats) < 0)
+                return -errno;
+        network->timestamp = timespec_load(&stats.st_mtim);
+
+        if (network_verify(network) < 0)
+                return 0;
```

Comment:
Why does this return 0? I know this code is just moved, but maybe add a comment explaining what is supposed to happen here.